.eslintrc.js

```diff
@@ -38,6 +44,13 @@ module.exports = {
     '@typescript-eslint/no-use-before-define': 0,
     '@typescript-eslint/no-var-requires': 0,
 
+    // eslint-plugin-import
+    'import/namespace': [2, { allowComputed: true }],
+    'import/order': ['error', {
+      groups: ['builtin', 'external', 'internal', 'sibling', 'parent', 'index', 'object', 'type'],
+      pathGroups: [{ pattern: 'mock-utils', group: 'builtin', patternOptions: { matchBase: true, nocomment: true } }],
```

Comment:
`mockEthers` needs to be at the top of some test files so this adds an exception for the `mock-utils.ts` file